### PR TITLE
feat: add sensor config

### PR DESCRIPTION
**Pull request by ozym**
This builds a sensor config file in either XML or json format. It documents the sensors and their locations as well as manually collection sites. This updates the JSON part.

### CODE REVIEW

File: cmd/sensor-config/main.go [hunk 1]
```diff
@@ -125,12 +127,26 @@ func main() {
 		}
 		defer file.Close()
 
-		if err := network.MarshalIndent(file, "", "  "); err != nil {
-			log.Fatalf("unable to marshal output file %q: %v", settings.output, err)
+		switch {
+		case settings.json:
+			if err := network.EncodeJSON(file); err != nil {
+				log.Fatalf("unable to marshal output: %v", err)
```
Comment (wilsonjord) [inline cmd/sensor-config/main.go:133]: question: why not same log as the `default` case?

File: cmd/sensor-config/README.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
+# sensor-config
+
+Build a sensor description file
+
+## output
+
+The output XML or JSON file provides a description of all sensor installations and manual collection points.
```
Comment (wilsonjord) [inline cmd/sensor-config/README.md:7]: suggestion(non-blocking): `The output XML (default) or JSON file provides...`

**Review by wilsonjord — APPROVED** [review]

File: cmd/sensor-config/main.go (hunk 1 above)
Comment (ozym) [reply cmd/sensor-config/main.go]: just being copy/paste missing bits

File: cmd/sensor-config/README.md (hunk 2 above)
Comment (ozym) [reply cmd/sensor-config/README.md]: looks good.

**Pull request merged**